react-select.d.ts

```diff
@@ -0,0 +1,15 @@
+import { ForwardedRef, HTMLAttributes } from "react"
+import { Props } from "react-select"
+
+declare module "react-select" {
```

Comment:
Could we contribute on the @types/react-select instead ?

Reply:
major v5 is coming with a [large typescript rewrite](https://github.com/JedWatson/react-select/issues/4559)...
![image](https://user-images.githubusercontent.com/62407/134131328-874c39e0-fa56-4547-bd46-dbd7dae2e603.png)
I think the patch will not be necessary and some parts of the typings will need to be adapted.